INN/umbrella-mstoday
# Reduce h1 heading size on mobile for blank page template

## Changes

This pull request makes the following changes:

- Updates the sizing of `h1` elements on screens under 768px to be `38px` **for only the `Blank Page (MS Today special)` template**

## Why



For #75

## Testing/Questions

Features that this PR affects:

- Blank page template heading styles


Questions that need to be answered before merging:

- [ ] Are we sure we only want this change on the `Blank Page (MS Today special)` template?

Steps to test this PR:

1. Find a post using the `Blank Page (MS Today special)` template and add an H1 element to the body. View it on mobile and make sure its font size is 38px.
